packages/server/src/worker/mq.ts

```diff
@@ -118,6 +120,20 @@ async function addJobCheckNewAlerts(
   );
 }
 
+async function addJobDestroyAllSessions(
+  name: string = getName("-"),
+  userId
+): Promise<Job<TJobDestroyAllSessionsData, TJobDestroyAllSessionsResult>> {
```

Comment:
These job creators are looking pretty repetitive. Would it be better to create an `addJob` utility that used generic types instead?

Reply:
what would it look like? `addJob` then you put the job name and then it makes a switch statement to customize inputs? Seems about the same. I tend to be ok with copy-paste until a beneficial abstraction pops out